android/gradle.properties

```diff
@@ -1,6 +1,6 @@
-AppLovinMAX_compileSdkVersion=28
-AppLovinMAX_buildToolsVersion=28.0.3
-AppLovinMAX_targetSdkVersion=28
+ApplovinMax_minSdkVersion=21
+ApplovinMax_targetSdkVersion=31
+ApplovinMax_compileSdkVersion=31
```

Comment:
Should we update target and compile to be 34 as well? Actually, do you know where these are used?

Reply:
Ok, bump to 34.   Both `build.gradle` and `gradle.properties` are installed as a part of our package under the user's  `node_modules/node_modules/react-native-applovin-max/android/`. 

Here is a list of files in the new package.

```
$ npm publish --dry-run
npm notice
npm notice 📦  react-native-applovin-max@6.5.0
npm notice === Tarball Contents ===
npm notice 1.1kB  LICENSE
npm notice 533B   README.md
npm notice 2.1kB  android/build.gradle
npm notice 258B   android/gradle.properties
npm notice 127B   android/src/main/AndroidManifest.xml
npm notice 82B    android/src/main/AndroidManifestNew.xml
npm notice 11.2kB android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java
npm notice 4.5kB  android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java
npm notice 98.5kB android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java
npm notice 19.4kB android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java
npm notice 4.9kB  android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdViewManager.java
npm notice 1.1kB  android/src/main/java/com/applovin/reactnative/AppLovinMAXPackage.java
npm notice 2.0kB  ios/AppLovinMAX.h
npm notice 89.3kB ios/AppLovinMAX.m
npm notice 355B   ios/AppLovinMAXAdView.h
npm notice 9.1kB  ios/AppLovinMAXAdView.m
npm notice 288B   ios/AppLovinMAXAdViewManager.h
npm notice 1.9kB  ios/AppLovinMAXAdViewManager.m
npm notice 460B   ios/AppLovinMAXNativeAdView.h
npm notice 12.7kB ios/AppLovinMAXNativeAdView.m
npm notice 338B   ios/AppLovinMAXNativeAdViewManager.h
npm notice 2.0kB  ios/AppLovinMAXNativeAdViewManager.m
npm notice 9.9kB  lib/commonjs/AdView.js
npm notice 10.0kB lib/commonjs/AdView.js.map
npm notice 2.0kB  lib/commonjs/AppLovinMAX.js
npm notice 1.2kB  lib/commonjs/AppLovinMAX.js.map
npm notice 3.7kB  lib/commonjs/AppOpenAd.js
npm notice 3.9kB  lib/commonjs/AppOpenAd.js.map
npm notice 4.6kB  lib/commonjs/BannerAd.js
npm notice 5.3kB  lib/commonjs/BannerAd.js.map
npm notice 981B   lib/commonjs/EventEmitter.js
npm notice 1.0kB  lib/commonjs/EventEmitter.js.map
npm notice 5.9kB  lib/commonjs/index.js
npm notice 2.3kB  lib/commonjs/index.js.map
npm notice 3.8kB  lib/commonjs/InterstitialAd.js
npm notice 3.9kB  lib/commonjs/InterstitialAd.js.map
npm notice 3.8kB  lib/commonjs/MRecAd.js
npm notice 4.3kB  lib/commonjs/MRecAd.js.map
npm notice 5.8kB  lib/commonjs/nativeAd/NativeAdView.js
npm notice 5.3kB  lib/commonjs/nativeAd/NativeAdView.js.map
npm notice 8.3kB  lib/commonjs/nativeAd/NativeAdViewComponents.js
npm notice 9.0kB  lib/commonjs/nativeAd/NativeAdViewComponents.js.map
npm notice 1.8kB  lib/commonjs/nativeAd/NativeAdViewProvider.js
npm notice 2.3kB  lib/commonjs/nativeAd/NativeAdViewProvider.js.map
npm notice 286B   lib/commonjs/Privacy.js
npm notice 299B   lib/commonjs/Privacy.js.map
npm notice 4.2kB  lib/commonjs/RewardedAd.js
npm notice 4.3kB  lib/commonjs/RewardedAd.js.map
npm notice 5.8kB  lib/commonjs/TargetingData.js
npm notice 4.8kB  lib/commonjs/TargetingData.js.map
npm notice 115B   lib/commonjs/types/AdEvent.js
npm notice 113B   lib/commonjs/types/AdEvent.js.map
npm notice 1.3kB  lib/commonjs/types/AdInfo.js
npm notice 462B   lib/commonjs/types/AdInfo.js.map
npm notice 115B   lib/commonjs/types/AdProps.js
npm notice 113B   lib/commonjs/types/AdProps.js.map
npm notice 119B   lib/commonjs/types/AdViewProps.js
npm notice 117B   lib/commonjs/types/AdViewProps.js.map
npm notice 119B   lib/commonjs/types/AppLovinMAX.js
npm notice 117B   lib/commonjs/types/AppLovinMAX.js.map
npm notice 117B   lib/commonjs/types/AppOpenAd.js
npm notice 115B   lib/commonjs/types/AppOpenAd.js.map
npm notice 116B   lib/commonjs/types/BannerAd.js
npm notice 114B   lib/commonjs/types/BannerAd.js.map
npm notice 116B   lib/commonjs/types/CMPError.js
npm notice 114B   lib/commonjs/types/CMPError.js.map
npm notice 121B   lib/commonjs/types/Configuration.js
npm notice 119B   lib/commonjs/types/Configuration.js.map
npm notice 120B   lib/commonjs/types/FullscreenAd.js
npm notice 118B   lib/commonjs/types/FullscreenAd.js.map
npm notice 1.9kB  lib/commonjs/types/index.js
npm notice 1.8kB  lib/commonjs/types/index.js.map
npm notice 122B   lib/commonjs/types/InterstitialAd.js
npm notice 120B   lib/commonjs/types/InterstitialAd.js.map
npm notice 114B   lib/commonjs/types/MRecAd.js
npm notice 112B   lib/commonjs/types/MRecAd.js.map
npm notice 50B    lib/commonjs/types/NativeAd.js
npm notice 114B   lib/commonjs/types/NativeAd.js.map
npm notice 125B   lib/commonjs/types/NativeAdViewProps.js
npm notice 123B   lib/commonjs/types/NativeAdViewProps.js.map
npm notice 49B    lib/commonjs/types/Privacy.js
npm notice 113B   lib/commonjs/types/Privacy.js.map
npm notice 118B   lib/commonjs/types/RewardedAd.js
npm notice 116B   lib/commonjs/types/RewardedAd.js.map
npm notice 121B   lib/commonjs/types/TargetingData.js
npm notice 119B   lib/commonjs/types/TargetingData.js.map
npm notice 114B   lib/commonjs/types/ViewAd.js
npm notice 112B   lib/commonjs/types/ViewAd.js.map
npm notice 8.8kB  lib/module/AdView.js
npm notice 8.4kB  lib/module/AdView.js.map
npm notice 1.7kB  lib/module/AppLovinMAX.js
npm notice 1.0kB  lib/module/AppLovinMAX.js.map
npm notice 3.3kB  lib/module/AppOpenAd.js
npm notice 3.7kB  lib/module/AppOpenAd.js.map
npm notice 4.2kB  lib/module/BannerAd.js
npm notice 5.1kB  lib/module/BannerAd.js.map
npm notice 750B   lib/module/EventEmitter.js
npm notice 949B   lib/module/EventEmitter.js.map
npm notice 769B   lib/module/index.js
npm notice 962B   lib/module/index.js.map
npm notice 3.3kB  lib/module/InterstitialAd.js
npm notice 3.7kB  lib/module/InterstitialAd.js.map
npm notice 3.4kB  lib/module/MRecAd.js
npm notice 4.2kB  lib/module/MRecAd.js.map
npm notice 4.8kB  lib/module/nativeAd/NativeAdView.js
npm notice 3.7kB  lib/module/nativeAd/NativeAdView.js.map
npm notice 6.3kB  lib/module/nativeAd/NativeAdViewComponents.js
npm notice 6.9kB  lib/module/nativeAd/NativeAdViewComponents.js.map
npm notice 807B   lib/module/nativeAd/NativeAdViewProvider.js
npm notice 1.1kB  lib/module/nativeAd/NativeAdViewProvider.js.map
npm notice 158B   lib/module/Privacy.js
npm notice 240B   lib/module/Privacy.js.map
npm notice 3.7kB  lib/module/RewardedAd.js
npm notice 4.1kB  lib/module/RewardedAd.js.map
npm notice 5.6kB  lib/module/TargetingData.js
npm notice 4.7kB  lib/module/TargetingData.js.map
npm notice 46B    lib/module/types/AdEvent.js
npm notice 113B   lib/module/types/AdEvent.js.map
npm notice 1.2kB  lib/module/types/AdInfo.js
npm notice 434B   lib/module/types/AdInfo.js.map
npm notice 46B    lib/module/types/AdProps.js
npm notice 113B   lib/module/types/AdProps.js.map
npm notice 50B    lib/module/types/AdViewProps.js
npm notice 117B   lib/module/types/AdViewProps.js.map
npm notice 50B    lib/module/types/AppLovinMAX.js
npm notice 117B   lib/module/types/AppLovinMAX.js.map
npm notice 48B    lib/module/types/AppOpenAd.js
npm notice 115B   lib/module/types/AppOpenAd.js.map
npm notice 47B    lib/module/types/BannerAd.js
npm notice 114B   lib/module/types/BannerAd.js.map
npm notice 47B    lib/module/types/CMPError.js
npm notice 114B   lib/module/types/CMPError.js.map
npm notice 52B    lib/module/types/Configuration.js
npm notice 119B   lib/module/types/Configuration.js.map
npm notice 51B    lib/module/types/FullscreenAd.js
npm notice 118B   lib/module/types/FullscreenAd.js.map
npm notice 188B   lib/module/types/index.js
npm notice 193B   lib/module/types/index.js.map
npm notice 53B    lib/module/types/InterstitialAd.js
npm notice 120B   lib/module/types/InterstitialAd.js.map
npm notice 45B    lib/module/types/MRecAd.js
npm notice 112B   lib/module/types/MRecAd.js.map
npm notice 37B    lib/module/types/NativeAd.js
npm notice 114B   lib/module/types/NativeAd.js.map
npm notice 56B    lib/module/types/NativeAdViewProps.js
npm notice 123B   lib/module/types/NativeAdViewProps.js.map
npm notice 36B    lib/module/types/Privacy.js
npm notice 113B   lib/module/types/Privacy.js.map
npm notice 49B    lib/module/types/RewardedAd.js
npm notice 116B   lib/module/types/RewardedAd.js.map
npm notice 52B    lib/module/types/TargetingData.js
npm notice 119B   lib/module/types/TargetingData.js.map
npm notice 45B    lib/module/types/ViewAd.js
npm notice 112B   lib/module/types/ViewAd.js.map
npm notice 1.9kB  lib/typescript/src/AdView.d.ts
npm notice 597B   lib/typescript/src/AdView.d.ts.map
npm notice 2.0kB  lib/typescript/src/AppLovinMAX.d.ts
npm notice 660B   lib/typescript/src/AppLovinMAX.d.ts.map
npm notice 168B   lib/typescript/src/AppOpenAd.d.ts
npm notice 223B   lib/typescript/src/AppOpenAd.d.ts.map
npm notice 162B   lib/typescript/src/BannerAd.d.ts
npm notice 221B   lib/typescript/src/BannerAd.d.ts.map
npm notice 298B   lib/typescript/src/EventEmitter.d.ts
npm notice 291B   lib/typescript/src/EventEmitter.d.ts.map
npm notice 772B   lib/typescript/src/index.d.ts
npm notice 746B   lib/typescript/src/index.d.ts.map
npm notice 198B   lib/typescript/src/InterstitialAd.d.ts
npm notice 236B   lib/typescript/src/InterstitialAd.d.ts.map
npm notice 150B   lib/typescript/src/MRecAd.d.ts
npm notice 217B   lib/typescript/src/MRecAd.d.ts.map
npm notice 1.6kB  lib/typescript/src/nativeAd/NativeAdView.d.ts
npm notice 349B   lib/typescript/src/nativeAd/NativeAdView.d.ts.map
npm notice 772B   lib/typescript/src/nativeAd/NativeAdViewComponents.d.ts
npm notice 623B   lib/typescript/src/nativeAd/NativeAdViewComponents.d.ts.map
npm notice 759B   lib/typescript/src/nativeAd/NativeAdViewProvider.d.ts
npm notice 795B   lib/typescript/src/nativeAd/NativeAdViewProvider.d.ts.map
npm notice 132B   lib/typescript/src/Privacy.d.ts
npm notice 197B   lib/typescript/src/Privacy.d.ts.map
npm notice 174B   lib/typescript/src/RewardedAd.d.ts
npm notice 225B   lib/typescript/src/RewardedAd.d.ts.map
npm notice 708B   lib/typescript/src/TargetingData.d.ts
npm notice 420B   lib/typescript/src/TargetingData.d.ts.map
npm notice 666B   lib/typescript/src/types/AdEvent.d.ts
npm notice 477B   lib/typescript/src/types/AdEvent.d.ts.map
npm notice 8.3kB  lib/typescript/src/types/AdInfo.d.ts
npm notice 3.0kB  lib/typescript/src/types/AdInfo.d.ts.map
npm notice 2.2kB  lib/typescript/src/types/AdProps.d.ts
npm notice 1.0kB  lib/typescript/src/types/AdProps.d.ts.map
npm notice 1.4kB  lib/typescript/src/types/AdViewProps.d.ts
npm notice 711B   lib/typescript/src/types/AdViewProps.d.ts.map
npm notice 4.8kB  lib/typescript/src/types/AppLovinMAX.d.ts
npm notice 1.6kB  lib/typescript/src/types/AppLovinMAX.d.ts.map
npm notice 141B   lib/typescript/src/types/AppOpenAd.d.ts
npm notice 213B   lib/typescript/src/types/AppOpenAd.d.ts.map
npm notice 1.6kB  lib/typescript/src/types/BannerAd.d.ts
npm notice 733B   lib/typescript/src/types/BannerAd.d.ts.map
npm notice 440B   lib/typescript/src/types/CMPError.d.ts
npm notice 356B   lib/typescript/src/types/CMPError.d.ts.map
npm notice 1.4kB  lib/typescript/src/types/Configuration.d.ts
npm notice 416B   lib/typescript/src/types/Configuration.d.ts.map
npm notice 4.9kB  lib/typescript/src/types/FullscreenAd.d.ts
npm notice 1.7kB  lib/typescript/src/types/FullscreenAd.d.ts.map
npm notice 190B   lib/typescript/src/types/index.d.ts
npm notice 226B   lib/typescript/src/types/index.d.ts.map
npm notice 151B   lib/typescript/src/types/InterstitialAd.d.ts
npm notice 225B   lib/typescript/src/types/InterstitialAd.d.ts.map
npm notice 423B   lib/typescript/src/types/MRecAd.d.ts
npm notice 338B   lib/typescript/src/types/MRecAd.d.ts.map
npm notice 930B   lib/typescript/src/types/NativeAd.d.ts
npm notice 603B   lib/typescript/src/types/NativeAd.d.ts.map
npm notice 323B   lib/typescript/src/types/NativeAdViewProps.d.ts
npm notice 322B   lib/typescript/src/types/NativeAdViewProps.d.ts.map
npm notice 1.3kB  lib/typescript/src/types/Privacy.d.ts
npm notice 525B   lib/typescript/src/types/Privacy.d.ts.map
npm notice 703B   lib/typescript/src/types/RewardedAd.d.ts
npm notice 435B   lib/typescript/src/types/RewardedAd.d.ts.map
npm notice 1.6kB  lib/typescript/src/types/TargetingData.d.ts
npm notice 818B   lib/typescript/src/types/TargetingData.d.ts.map
npm notice 5.3kB  lib/typescript/src/types/ViewAd.d.ts
npm notice 1.8kB  lib/typescript/src/types/ViewAd.d.ts.map
npm notice 3.8kB  package.json
npm notice 1.8kB  react-native-applovin-max.podspec
npm notice 10.1kB src/AdView.tsx
npm notice 2.5kB  src/AppLovinMAX.ts
npm notice 4.1kB  src/AppOpenAd.ts
npm notice 5.2kB  src/BannerAd.ts
npm notice 983B   src/EventEmitter.ts
npm notice 769B   src/index.ts
npm notice 4.2kB  src/InterstitialAd.ts
npm notice 4.3kB  src/MRecAd.ts
npm notice 6.1kB  src/nativeAd/NativeAdView.tsx
npm notice 6.2kB  src/nativeAd/NativeAdViewComponents.tsx
npm notice 1.4kB  src/nativeAd/NativeAdViewProvider.tsx
npm notice 188B   src/Privacy.ts
npm notice 4.6kB  src/RewardedAd.ts
npm notice 7.5kB  src/TargetingData.ts
npm notice 627B   src/types/AdEvent.ts
npm notice 8.3kB  src/types/AdInfo.ts
npm notice 2.1kB  src/types/AdProps.ts
npm notice 1.4kB  src/types/AdViewProps.ts
npm notice 4.8kB  src/types/AppLovinMAX.ts
npm notice 103B   src/types/AppOpenAd.ts
npm notice 1.6kB  src/types/BannerAd.ts
npm notice 406B   src/types/CMPError.ts
npm notice 1.4kB  src/types/Configuration.ts
npm notice 4.8kB  src/types/FullscreenAd.ts
npm notice 155B   src/types/index.ts
npm notice 108B   src/types/InterstitialAd.ts
npm notice 388B   src/types/MRecAd.ts
npm notice 900B   src/types/NativeAd.ts
npm notice 278B   src/types/NativeAdViewProps.ts
npm notice 1.3kB  src/types/Privacy.ts
npm notice 665B   src/types/RewardedAd.ts
npm notice 1.6kB  src/types/TargetingData.ts
npm notice 5.3kB  src/types/ViewAd.ts
npm notice === Tarball Details ===
npm notice name:          react-native-applovin-max
npm notice version:       6.5.0
npm notice filename:      react-native-applovin-max-6.5.0.tgz
npm notice package size:  106.5 kB
npm notice unpacked size: 658.9 kB
npm notice shasum:        672b44fd3ff3275081ebe4c8c77674c1760c1d95
npm notice integrity:     sha512-xaMNqKu5IiHLh[...]GzBBUZIfEhlvA==
npm notice total files:   255
npm notice
+ react-native-applovin-max@6.5.0
```